ebscopy/ebscopy.py

```diff
@@ -240,7 +242,7 @@ def _change_eds_date_limiter_to_api(l):
 	# 1-19780515	(bad start)
 	# 19780515-1	(bad end)
 	#
-	if l["Id"] != "DT1" or not re.search("\A\s*(?=-?\d{8})(?:\d{8})?-(?:\d{8})?\Z", l["Values"][0]):
+	if l["Id"] != "DT1" or not re.search("\\A\\s*(?=-?\\d{8})(?:\\d{8})?-(?:\\d{8})?\\Z", l["Values"][0]):
```

Comment:
Is this a fix, a change due to version, or just a linter preference?

Reply:
Linting picked this up - adding the additional backslash cleared the "Anomalous backslash in string: '\A'. String constant might be missing an r prefix." warning.

Reply:
I think I need to review this concept - I think the additional backslash is escaping the character entirely.

Reply:
Yeah, I don't trust the linter on complex regular expressions. We might have to simplify the regular expression or definite it more cleanly.